packages/puppeteer-core/src/cdp/ExecutionContext.ts

```diff
@@ -59,10 +60,17 @@ const ariaQuerySelectorAllBinding = new Binding(
   }) as (...args: unknown[]) => unknown
 );
 
+interface ExecutionContextEvents extends Record<EventType, unknown> {
+  dispose: undefined;
```

Comment:
Do we need to have an Enum for events?

Reply:
We can do what we do BiDi core, though it would be nice to only allow that for internal classes.

Reply:
Nit:
```suggestion
  disposed: undefined;
```